libpromises/cf3lex.l

```diff
@@ -135,20 +135,23 @@ varclass       (\"[^"]*\"|\'[^']*\')::
 promise_type   [a-zA-Z_]+:
 
 %%
-
-{line}                {
+{eat_line}             {
```

Comment:
Can you add a comment here, that explains that `eat_line` is only used to store the line for error reporting, and all line is returned back and re-lexed ?